sky130-build/build.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python3
+
+import argparse
+import copy
+import siliconcompiler
+import os
+
+from sources import add_sources
+from libs import lib_setup
+
+def setup_options(chip):
+        chip.set('option', 'define', 'VERILATOR')
+        chip.set('option', 'loglevel', 'INFO')
+        chip.set('option', 'relax', True)
+        chip.set('option', 'jobname', 'job0')
+
+        chip.add('option', 'define', 'WT_DCACHE')
+        
+        cur_dir = os.path.dirname(os.path.realpath(__file__))
+        chip.add('option', 'define', f'MEM_ROOT={cur_dir}')
+
+def configure_chip(remote=False):
+        chip = siliconcompiler.Chip('cva6')
+
+        setup_options(chip)
+
+        chip.set('option', 'frontend', 'systemverilog')
+        chip.set('option', 'cmdfile', 'design.f')
+        chip.load_target('skywater130_demo')
+
+        chip.set('tool', 'openroad', 'var', 'place', '0', 'place_density', ['0.15'])
+        chip.set('tool', 'openroad', 'var', 'route', '0', 'grt_allow_congestion', ['true'])
+
+        chip.set('asic', 'macrolib', ['sky130sram'])
+        lib_setup(chip)
+
+        for step in ('extspice', 'drc'):
+                chip.set('tool', 'magic', 'var', step, '0', 'exclude', ['sky130sram'])
+        chip.set('tool', 'netgen', 'var', 'lvs', '0', 'exclude', ['sky130sram'])
+
+        add_sources(chip)
+
+        chip.clock('we_din\[5\]', period=20)
```

Comment:
`'we_din\[5\]'` is referring to one of the ZeroSoC pins. If you want to constrain the timing, you could change this to `'clk_i'`. (this would set up a 20ns clock by default, it might take some playing around with that number to see if the design meets timing though -- if it does, the OpenROAD step will report a setupslack and holdslack of 0 or more).